### DISCUSSION TITLE
Question settings RER :)

### DISCUSSION THREAD

**Discussion by armandpesme** [Aelto/W3_RandomEncounters_Tweaks#233] [Q&A]
I apologize for my repeated questions, it's because I have the greatestadmiration and passion for this mod.
I don't speak English well, and I can't find the following option thatallows the difficulty selection to be done automatically:
“Allow selecting contracts difficulty. Default: false.”
I also can't find the setting to disable this:
“Reputation System enabled. Default: true.”
Nor can I find the option to disable reputation requirements:
“Reputation required for vanilla contracts. Default: 2.”
Does a particular seed exist that only generates boss contracts, notregular monster contracts?
“Contracts generation seed. Default: 0.”
I can't locate the setting to ensure the rewards are set to in-gamecurrency:
“Type of rewards. Default: Crowns.”
 
Thank you very much again! <3

**Comment by Aelto** [comment]
Hi, sorry for the delay the recent days were quite busy! No worries, feel free to ask any question related to the mod, it's fully understandable considering the size of the mod.

I believe you're asking about the settings that are listed on the RER Bible? It's possible they are outdated and they describe the old menu from the v3.3. The v3.4 update allowed me to refine the design around the contracts and so most of them were either removed or integrated directly into the game
- `Allow selecting contract difficulty` is not there anymore, instead there is an option to select & lock the difficulty after interacting with a board
- `Reputation system enabled`  was removed as well, the reputation system is now forced since you can freely select the difficulty at any board
- `Reputation required for vanilla contracts`, the option as well as the feature was removed from the mod
- `Contracts generation seed`, no there is no seed that results in boss fights only. But note that as your reputation increases the chances of getting smaller target decrease, you may just need to complete a few contracts to get there
- `Type of rewards` was removed, it used to offer a choice between crowns & tokens of gratitude. The v3.4 now defaults to crowns and there is no way to get tokens anymore (they're still in the mod in case you had some however)

Hope this helps! And I'll note that I should probably update the RER Bible!

**Comment by armandpesme** [comment]
No, I actually think it's quite fast. Thank you so much for taking the time to answer each question, it really helped me a lot!
And by the way, your mod is amazing!
